index.html

```diff
@@ -207,14 +207,48 @@
                     <span class="mif-file-download mif-3x"></span>
                     <span class="caption">Sequence file</span>
                 </button>
-                <button class="ribbon-button" onclick="view.toggleWindow('videoCreationWindow')">
-                    <span class="mif-video-camera mif-3x"></span>
-                    <span class="caption">Create video</span>
-                </button>
                 <button class="ribbon-button" onclick="view.saveCanvasImage()" title="Save current view as an image ('p')">
                     <span class="mif-photo-camera mif-3x"></span>
                     <span class="caption">Save image</span>
                 </button>
+                <button id="videoCreateButton" class="ribbon-button" onclick="view.toggleWindow('videoCreationWindow')" disabled="true">
```

Comment:
You still want to be able to create "lemniscate" videos if you don't have a trajectory

Reply:
seems this function is not working on main as well ?